src/scripts/components/stories/pilot-story/provider/chapter-provider.tsx

```diff
@@ -0,0 +1,65 @@
+import React, {
+  FunctionComponent,
+  PropsWithChildren,
+  createContext,
+  useEffect,
+  useState
+} from 'react';
+import {useHistory} from 'react-router-dom';
+import {useParallaxController} from 'react-scroll-parallax';
+import useIntersectChapters from '../hooks/use-chapter-observer';
+import {chapterMainElement} from '../config/main';
+
+function useChapterContext() {
```

Comment:
Nice! I think I will refactor globe component to use this hook for obtaining chapter progress.

Reply:
cool!